src/scd.cpp

```diff
@@ -723,6 +742,42 @@ namespace openre::scd
         return SCD_RESULT_NEXT;
     }
 
+    // 0x004E52E0
+    static int scd_item_aot_set(SceTask* sce)
+    {
+        auto opcode = reinterpret_cast<SceItemAotSet*>(sce->data);
+        sce->data += sizeof(SceItemAotSet);
+        if (!gGameTable.aot_table[opcode->aot_id])
+        {
+            gGameTable.aot_count++;
+        }
+        gGameTable.aot_table[opcode->aot_id] = &opcode->sce;
+        auto flagGroup = gGameTable.current_stage < 4 ? FlagGroup::Item : FlagGroup::Item2;
+        // Item already picked up
+        if (check_flag(flagGroup, opcode->flag))
+        {
+            auto aot = (uint32_t*)(gGameTable.aot_table[opcode->aot_id]);
```

Comment:
You should already have the reference to this earlier, which I guess is just &opcode->sce.